Pipfile

```diff
@@ -16,7 +16,6 @@ requests = "*"  # Needed by plotly's image exporter.
 psutil = "*"  # Needed by plotly's image exporter.
 pandas = "*"  # Needed by plotly's express library.
 geopandas = "*"
-matplotlib = "*"  # Needed by geopandas to plot maps.
```

Comment:
Did you mean to delete this (great if we don't need to declare this explicitly, but from memory I don't think that worked for me...)?

Reply:
That was an editing error. thanks for catching it added back